checkuser

```diff
@@ -248,7 +261,7 @@ else
 fi
 
 echo -n "Checking whether user has a scratch directory: "
-if stat --printf='' "/scratch/scratch/${username}" 2>/dev/null
+if stat --printf='' "${user_scratch_dir}" 2>/dev/null
```

Comment:
This now prints incorrect results on Myriad, because previously it was possible to stat anyone's scratch directory regardless of who you were and now it isn't.